### ISSUE TITLE
SyntaxError: Non-ASCII character '\xc3'

### ISSUE THREAD

**Issue by Yaswanth-jami** [RUB-NDS/PRET#48]
When I am trying to run the code using python 2.7
It's showing the following error

File "pret.py", line 38
SyntaxError: Non-ASCII character '\xc3' in file pret.py on line 38, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

**Comment by Yaswanth-jami** [comment]
Does anyone know how to solve this issue?

**Comment by MrR3boot** [comment]
Run it with python3

**Comment by Yaswanth-jami** [comment]
> Run it with python3

when I run it with python3, the device is getting connected but the commands are not working.
It's showing  
Command execution failed (a bytes-like object is required, not 'str')

**Comment by MrR3boot** [comment]
Yup i've raised an issue on this already. Check [49](https://github.com/RUB-NDS/PRET/issues/49)

**Comment by MrR3boot** [comment]
You better use [these](https://github.com/RUB-NDS/PRET/tree/4f3820a83cd0e8bd88fcc1d09641e54720e5bbc9) files.

**Comment by dpcxpert** [comment]
just for more details this is what i got when testing my and my friends printers using a bash script it said something is wrong with printer.py and pret.py
![issuepng_scrubbed](https://user-images.githubusercontent.com/39003214/122592421-26b69280-d05c-11eb-928c-ae2136ebb432.png)
